Makefile

```diff
@@ -1,7 +1,5 @@
 #
-# synse-client-go
-#
-
+# synse-client-go # 
```

Comment:
why the change in the header comment?

Reply:
it was a mistake! thanks for catching this.